README.rst

```diff
@@ -86,6 +86,18 @@ Registry
               to:
                 - name@receivehost.com
 
+Docker login to private registry
+--------------------------------
+
+.. code-block:: yaml
+
+    docker:
+      host:
+        enabled: true
+        registry:
```

Comment:
If I am not wrong, there can be multiple logged in registries. So this metadata should be a list.